tests/dummy/app/components/menu-link-item.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import layout from '../templates/components/menu-link-item';
+
+const { Component } = Ember;
```

Comment:
Here's a great reference for updating the code in this PR to use shims: https://github.com/ember-cli/ember-cli-shims/blob/master/app-shims.js